lib/cisco_node_utils/version.rb

```diff
@@ -14,5 +14,5 @@
 
 # Container module for version number only.
 module CiscoNodeUtils
-  VERSION = '1.0.2-dev'
+  VERSION = '1.0.2'
```

Comment:
This change is unrelated to PR. But it also drives me nuts because I have to change it to get the gem to build; want to submit it in a separate PR?